crates/miden-lib/src/note/well_known_note.rs

```diff
@@ -336,71 +336,72 @@ impl WellKnownNote {
 // HELPER FUNCTIONS
 // ================================================================================================
 
-/// Returns the receiver account ID parsed from the provided P2ID note inputs.
+/// Returns the receiver account ID parsed from the provided P2ID check that note storage has
+/// correct length..
```

Comment:
```suggestion
/// Returns the receiver account ID parsed from the provided P2ID note storage.
```

I think this is sufficient.